fuzzbench/test_e2e_run.py

```diff
@@ -41,3 +45,12 @@ def test_db_contains_experiment_results(self):
     def test_experiment_report_is_generated(self):
         """Fake test to be implemented later."""
         assert True
+
+    def test_default_config_is_used(self):
+        """Tests whether the default config is used for end to end test."""
+        try:
+            fake_job = Job.fetch('e2e-test',
+                                 connection=redis.Redis(host='queue-server'))
+        except rq.exceptions.NoSuchJobError:
```

Comment:
No need of try except, exception raised in test is perfectly fine and is a failure.

Reply:
This is cool!